### DISCUSSION TITLE
Open Bag Testing Updates

### DISCUSSION THREAD

**Discussion by RugBark** [frc5567/2023-Robot#161] [Team Posts]
After working on the unbagged robot today this is the list we wrote of changes that we need to have done:
• Create pickup height (~2-3 inches above Level 1, servo at an angle ~45 degrees)
The pickup height will be assigned to the button on the gamepad that Level Zero is currently on
• m_elevator is instantiated twice in Robot, one of those needs to be deleted
• testContinuousCommand can be removed since we are done using that method
• Any references to DashboardData and CustomDashboard need to be removed as it is currently interfering with our camera streams.

**Comment by MikasaSucasa** [comment]
We didn't flash the firmware on the Rio... priority number 1 now
